src/modules/app/tasks/tasks.component.ts

```diff
@@ -13,6 +13,22 @@ export class TasksComponent implements OnInit {
   public error: string = '';
   public user: any = {};
   public tasks: Task[] = [];
+  public newTasks: Task[] = [];
+  public inProcessTasks: Task[] = [];
+  public doneTasks: Task[] = [];
+  public currentDelTask: any = {};
```

Comment:
wierd

Reply:
Unfortunately, these variables play important role in the logic of my code an i have no idea to way how remove them.